src/tooling/piral-cli/src/apps/debug-pilet.ts

```diff
@@ -204,6 +204,7 @@ export async function debugPilet(baseDir = process.cwd(), options: DebugPiletOpt
   }
 
   const injectorConfig = {
+    ssl: krasConfig.ssl,
```

Comment:
I don't think thats necessary -> see below.

Reply:
Thanks for pointing right